src/Main.java

```diff
@@ -59,7 +61,10 @@ public void paintComponent(Graphics graphics) {
             super.paintComponent(graphics);
 
             //System.out.println("* Repaint *");
-
+            String probeersel = "joost";
+            if(probeersel == "joost"){
+                //hooooofoft;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)